packages/chat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carbon-labs/ai-chat",
-  "version": "0.4.0",
+  "version": "3.1.0-rc.1",
```

Comment:
No need to change the version here. When we publish, the github action will bump up the version for you.